types/buffer-crc32/buffer-crc32-tests.ts

```diff
@@ -1,19 +1,23 @@
 import crc32 = require('buffer-crc32');
```

Comment:
```suggestion
import crc32 from 'buffer-crc32';
```

Reply:
No, we have a commonjs export here, so the only way to import that'll work correctly everywhere (independent of tsconfig settings) is a commonjs import.